cmd/describe/cluster/cmd.go

```diff
@@ -743,10 +743,17 @@ Nodes:
 			)
 		}
 	}
+	hasSgsCompute := len(cluster.AWS().AdditionalComputeSecurityGroupIds()) > 0
 	hasSgsControlPlane := len(cluster.AWS().AdditionalControlPlaneSecurityGroupIds()) > 0
 	hasSgsInfra := len(cluster.AWS().AdditionalInfraSecurityGroupIds()) > 0
-	if hasSgsControlPlane || hasSgsInfra {
+	if hasSgsControlPlane || hasSgsInfra || hasSgsCompute {
 		nodeConfig += " - Additional Security Group IDs:\n"
+		if hasSgsCompute {
+			nodeConfig += fmt.Sprintf(
+				"   - Compute:		%s\n",
```

Comment:
Oh this is intentionally left out because the customer is supposed to check it from describe machine pool :)